pnpm/package.json

```diff
@@ -13,7 +13,8 @@
   "unpkg": "dist/pnpm.cjs",
   "files": [
     "dist",
-    "bin"
+    "bin",
+    "!*.map"
```

Comment:
isn't this only needed for local development?

Reply:
I have no problem developing locally with `pd` debug package under `@pnpm` scope, but the `pnpm` package it self don't seem to map to the source code.

if[ `entryPoints`  here set to `src/pnpm.ts` ](https://github.com/pnpm/pnpm/blob/main/pnpm/dev/pd.js#L47) then debug `pnpm` package itself have no problem for me also.

Reply:
yes, in my understanding, `!*.map` will exclude all files with the suffix `.map`, including `dist/pnpm.cjs.map`

Reply:
Sorry, I didn't notice that the `sourcemap` of esbuild has been enabled under `dev/scripts.js`. Now we have two ways to map to the source code on pnpm: 1. Enable the `sourcemap` under `tsconfig/compilerOptions`; 2. Adjust the esbuild entrypoints to `src/pnpm.ts`. which do you think is better @zkochan @await-ovo